src/sentry/api/endpoints/organization_release_assemble.py

```diff
@@ -69,19 +82,19 @@ def post(self, request: Request, organization, version) -> Response:
         if not chunks:
             return Response({"state": ChunkFileState.NOT_FOUND, "missingChunks": []}, status=200)
 
-        set_assemble_status(
-            AssembleTask.RELEASE_BUNDLE, organization.id, checksum, ChunkFileState.CREATED
-        )
+        set_assemble_status(assemble_task, organization.id, checksum, ChunkFileState.CREATED)
 
         from sentry.tasks.assemble import assemble_artifacts
 
         assemble_artifacts.apply_async(
             kwargs={
                 "org_id": organization.id,
                 "version": version,
+                "project_ids": project_ids,
+                "dist": None,  # TODO: a release can have multiple dists, which one is this?
```

Comment:
This confuses me a bit… how do release bundles work right now considering that an event sends a `dist`, but the bundle is uploaded without one?